lib/ChunkTemplate.js

```diff
@@ -12,6 +12,8 @@ function ChunkTemplate(outputOptions) {
 module.exports = ChunkTemplate;
 
 ChunkTemplate.prototype = Object.create(Template.prototype);
+ChunkTemplate.prototype.constructor = ChunkTemplate;
```

Comment:
I never used this kind of construct but let me explain my p.o.v. which might be totally wrong because i don't use this kind.
My logic says it will become an infinite recursion because:

When one tries to instantiate using ``new ChunkTemplate()``. It calls the  constructor of ``ChunkTemplate``, which turns out to be itself who calls it's own constructor in the process, which turns out to be itself etc. etc. ad-infinitum...

Like i said i might be totally wrong, correct me if so.
(That's why i remarked the comment as a question)